dev-tool.sh

```
@@ -179,20 +176,25 @@ branch() {
     git branch --show-current
 }
 
+# Initialize a new branch
+init() {
+    git checkout -b $1
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)